test/e2e/auth/service_accounts.go

```diff
@@ -153,6 +153,14 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
+	/*
+				Release: v1.9
+				Testname: Service Account Tokens Must AutoMount
+				Description: Ensure that Service Account keys are  mounted into the Pod. Pod contains
+		 three containers each will read Service Account Key, root CA Key and Namespace Key respectively from
```

Comment:
remove `Key` - that's just an artifact of code reuse: `... containers will each read the Service Account token, root CA, and  default namespace`

Reply:
fixed.